pootle/settings/40-apps.conf

```diff
@@ -129,3 +129,16 @@ POOTLE_SIGNUP_ENABLED = True
 # Path to function (string) which provides data to mark user's activity
 # chart with events
 POOTLE_REPORTS_MARK_FUNC = ''
+
+
+#
+# Scoring
+#
+
+POOTLE_SCORING_CONF = {
+    'EDIT_COEF': 5.0/7,
+    'REVIEW_COEF': 2.0/7,
+    'SUGG_COEF': 0.2,
+    'ANALYZE_COEF': 0.1,
+    'SIMILARITY_THRESHOLD': 0.5,
+}
```

Comment:
Adding a new setting would require adding docs for them and including it on the release notes.